app/services/WcoMetadataJavaMappingStrategy.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import javax.xml.bind.JAXBElement
+import services.mapping.MetaDataBuilder
+import uk.gov.hmrc.http.cache.client.CacheMap
+import wco.datamodel.wco.dec_dms._2.Declaration
+import wco.datamodel.wco.documentmetadata_dms._2.MetaData
+trait WcoMetadataJavaMappingStrategy extends WcoMetadataMappingStrategy {
```

Comment:
There should be empty line between imports and trait/classes etc.

It'll be more readable

Reply:
Comment Addressed!